product_cost_security/models/product_product.py

```diff
@@ -1,9 +1,20 @@
 # Copyright 2018 Sergio Teruel - Tecnativa <sergio.teruel@tecnativa.com>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
     standard_price = fields.Float(groups="product_cost_security.group_product_cost")
+    user_can_see_standard_price_uom = fields.Boolean(
```

Comment:
Why did you add this field? Is not used in the product.product view

Reply:
Its because the view is common for both product.product and product.template

Reply:
and the field was getting calculated based on product.template instead of product.product